patch-package
--- a/node_modules/foundation-sites/js/foundation.util.mediaQuery.js
+++ b/node_modules/foundation-sites/js/foundation.util.mediaQuery.js
@@ -63,6 +63,8 @@ let matchMedia = window.matchMedia || (function() {
   }
 })();
 
+window.matchMedia = matchMedia;
+
 var MediaQuery = {
   queries: [],
 
@@ -109,7 +111,7 @@ var MediaQuery = {
     var query = this.get(size);
 
     if (query) {
-      return matchMedia(query).matches;
+      return window.matchMedia(query).matches;
     }
 
     return false;
@@ -160,7 +162,7 @@ var MediaQuery = {
     for (var i = 0; i < this.queries.length; i++) {
       var query = this.queries[i];
 
-      if (matchMedia(query.value).matches) {
+      if (window.matchMedia(query.value).matches) {
         matched = query;
       }
     }
